### PR TITLE
[test](decimalv2) add some regression cases about decimalv2

### PR DESCRIPTION
**Pull request by mrhhsg**
## Proposed changes

* All cases' results are tested and passed with decimalv3 
* Cases about:
    1.  Calculation ( +, - , *, /)
    2. Kinds of predicates(<, >, =, <>, in, not in, is null, is not null)
    3. Load test(from csv and select into)
    4. Runtime filter
    5. Delete conditions
    6. Key columns(agg/duplicate/uniq model, distributed/partition, bitmap index...)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by mrhhsg** [comment]
run buildall

**Comment by mrhhsg** [comment]
run buildall

**Comment by mrhhsg** [comment]
run buildall

File: regression-test/suites/datatype_p0/decimalv2/test_decimalv2_common.groovy [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+    }
+
+    insert_data(table_dup)
+    insert_data(table_agg)
+    insert_data(table_uniq)
+    insert_data(table_dist)
+    // insert_data(table_part)
+
+    def run_predicate_test = { table_name, col, col_value, bitmap_col, bitmap_col_value, in_list ->
+        def query = """
```
Comment (yiguolei) [inline regression-test/suites/datatype_p0/decimalv2/test_decimalv2_common.groovy:227]: add  comment here to explain test which case

File: regression-test/suites/datatype_p0/decimalv2/test_decimalv2_common.groovy [hunk 2]
```diff
@@ -0,0 +1,411 @@
...
+            `decimal_value1` decimalv2(8, 5) NULL COMMENT "",
+            `decimal_value2` decimalv2(16, 5) NULL COMMENT "",
+            INDEX `idx_key1` (`decimal_value1`) USING BITMAP,
+            INDEX `idx_key2` (`decimal_value2`) USING BITMAP
+          ) ENGINE=OLAP
+          DUPLICATE KEY(`decimal_key1`, `decimal_key2`)
+          COMMENT "OLAP"
+          DISTRIBUTED BY HASH(`decimal_key1`, `decimal_key2`) BUCKETS 4
+          PROPERTIES (
+          "replication_allocation" = "tag.location.default: 1",
```
Comment (yiguolei) [inline regression-test/suites/datatype_p0/decimalv2/test_decimalv2_common.groovy:105]: disable auto compaction please

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by xinyiZzz — APPROVED** [review]
LGTM

**Pull request merged**